app/models/job.rb

```diff
@@ -14,6 +14,14 @@ class Job < ActiveRecord::Base
   default_scope { by_newest }
 
   def self.visible_for(user)
-    all.where.not(id: user.jobs_hidden.pluck(:id))
+    _not_hidden(user)._not_liked(user)
+  end
+
```

Comment:
Thats what the underscores are supposed to denote, I had them under `private` but hound was complaining about its uselessness